src/main/java/net/mcreator/ui/minecraft/states/block/JBlockPropertiesListEntry.java

```diff
@@ -126,35 +125,34 @@ public void setEntry(PropertyDataWithValue<?> prop) {
 		updatePropertyValues(prop);
 	}
 
-	private String getTypeString(PropertyData<?> data) {
-		return switch (data) {
-			case PropertyData.LogicType ignored -> "Logic";
-			case PropertyData.IntegerType ignored -> "Integer";
-			case null, default -> "Unknown";
-		};
-	}
-
 	private void updatePropertyValues(PropertyDataWithValue<?> data) {

```

Comment:
Do we still need this not too long method separately if it's only used once? (Maybe to split model and view actions)

Reply:
You can inline it.

Just don't do too many of those or we will delay this PR further

Reply:
Just this one and I don't see anything else (aside that next 2 PRs will have to be adjusted).